src/main/Config.cpp

```diff
@@ -240,6 +240,44 @@ readInt(ConfigItem const& item, T min = std::numeric_limits<T>::min(),
     }
     return static_cast<T>(v);
 }
+
+template <typename T>
+std::vector<T>
+readXdrEnumArray(ConfigItem const& item)
```

Comment:
This PR looks good to me, but I wanted to ask if you've tested filtering on a running node instead of just test cases? `readXdrEnumArray` looks fine to me, but isn't hit in our unit tests. Maybe we should add a simple unit test?

Reply:
Good idea, done.